posthog/cdp/test/test_site_functions.py

```diff
@@ -255,3 +262,24 @@ def test_get_transpiled_function_with_complex_filters(self):
         assert "const filterMatches = " in result
         assert '__getGlobal("event") == "$pageview"' in result
         assert "https://example.com" in result
+
+    def test_get_transpiled_function_with_mappings(self):
```

Comment:
Really would love to get proper testing working for these... I have the little node runner which is good for just seeing if there are compilation errors but I think we could likely mock out enough browser-y things to actually do proper tests via node at some point...

Reply:
Totally agreed. I'll try to think of something, but in a followup. Even node + jsdom would go a long way.